Samples/ResourceManagement/Compute/ConvertVirtualMachineToManagedDisks/ConvertVirtualMachineToManagedDisks.xproj

```diff
@@ -4,18 +4,16 @@
     <VisualStudioVersion Condition="'$(VisualStudioVersion)' == ''">14.0</VisualStudioVersion>
     <VSToolsPath Condition="'$(VSToolsPath)' == ''">$(MSBuildExtensionsPath32)\Microsoft\VisualStudio\v$(VisualStudioVersion)</VSToolsPath>
   </PropertyGroup>
-
   <Import Project="$(VSToolsPath)\DotNet\Microsoft.DotNet.Props" Condition="'$(VSToolsPath)' != ''" />
   <PropertyGroup Label="Globals">
-    <ProjectGuid>e52471d0-9127-483d-a326-5ce3b6b3aa86</ProjectGuid>
+    <ProjectGuid>d8d2bd1c-f9ef-405b-ab66-0cbb3fe6e559</ProjectGuid>
     <RootNamespace>MigrateVirtualMachine</RootNamespace>
```

Comment:
Confirming - shouldn't this be `<RootNamespace>ConvertVirtualMachineToManagedDisks</RootNamespace>`  ?